compiler/ballerina-lang/src/main/java/io/ballerina/compiler/api/symbols/SymbolKind.java

```diff
@@ -44,5 +44,6 @@ public enum SymbolKind {
     PARAMETER,
     PATH_PARAMETER,
     ANNOTATION_ATTACHMENT,
-    TUPLE_MEMBER
+    TUPLE_MEMBER,
+    PATH_SEGMENT
```

Comment:
Doesn't this correspond only to resource-path-segment-name? Why can't we use a more representative name?

Reply:
Yeah, better to have more narrower one. How about something like `PATH_NAME_SEGMENT`?